crates/primitives/starknet/src/transaction/utils.rs

```diff
@@ -0,0 +1,29 @@
+use alloc::collections::BTreeMap;
+use alloc::vec::Vec;
+
+use frame_support::BoundedVec;
+#[cfg(feature = "std")]
```

Comment:
make it no std

Reply:
it should yes

Reply:
@LucasLvy it is worth double check with Jonathan, but the last time I asked him, `starknet-core` is not yet `no-std` compatible. :/
And all the types in use here come from `starknet-core`.

Reply:
Once it's `no-std` compatible we can make the relevant changes here